google/generativeai/generative_models.py

```diff
@@ -275,13 +275,13 @@ def count_tokens(
         self, contents: content_types.ContentsType
     ) -> glm.CountTokensResponse:
         contents = content_types.to_contents(contents)
-        return self._client.count_tokens(self.model_name, contents)
+        return self._client.count_tokens(model=self.model_name, contents=contents)
 
     async def count_tokens_async(
         self, contents: content_types.ContentsType
     ) -> glm.CountTokensResponse:
         contents = content_types.to_contents(contents)
-        return await self._client.count_tokens(self.model_name, contents)
+        return await self._client.count_tokens(model=self.model_name, contents=contents)
```

Comment:
maybe here should be `self._async_client.count_tokens`

Reply:
I'm sending a PR for this, thanks.